.github/dependabot.yml

```diff
@@ -142,6 +143,30 @@ updates:
       - dependency-name: org.eclipse.microprofile.openapi:microprofile-openapi-tck
       - dependency-name: org.eclipse.microprofile.opentracing:microprofile-opentracing-tck*
       - dependency-name: org.eclipse.microprofile.rest.client:microprofile-rest-client-tck
+      - dependency-name: org.eclipse.microprofile.reactive-streams-operators:microprofile-reactive-streams-operators-*
```

Comment:
@cescoffier ?

Reply:
Except for the next update around Jakarta, this spec is almost dead. So, it can be removed.